titus-ext/eureka/src/main/java/com/netflix/titus/ext/eureka/containerhealth/EurekaContainerHealthService.java

```diff
@@ -194,13 +216,32 @@ private String takeStateReasonOf(Task task) {
         return Flux.fromIterable(events);
     }
 
-    private Optional<ContainerHealthEvent> handleTaskStateUpdate(Task task, ConcurrentMap<String, ContainerHealthEvent> state) {
+    private Flux<ContainerHealthEvent> handleJobEnabledStatusUpdate(Job current, Job previous, ConcurrentMap<String, ContainerHealthEvent> state) {
+
+        if (!JobFunctions.isServiceJob(current)) {
+            return Flux.empty();
+        }
+
+        // We need to examine in if a job 'enabled' status was changed.
```

Comment:
`We need to examine in if a job 'enabled' status was changed.` -> `Examine if a job's 'enabled' status was changed.`

Reply:
Fixed